docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+object store provider's documentation for more information, or, if using [Azure]
+blob storage, use the subsection
+[below](#internal-storage-configuration---azure-blob-storage) instead.
+
+For **local filesystems**, the only required field is a `url` parameter
+pointing to the target location - for example:
+
+```toml
+url = "file:///home/enduro/upload-dir"
+```
+
+**Default values**:
+
+```toml
+[internalStorage.bucket]
+endpoint = "http://minio.enduro-sdps:9000"
+pathStyle = true
+accessKey = "minio"
+secretKey = "minio123"
+region = "us-west-1"
+bucket = "internal"
+```
+
+* `endpoint`: API endpoint for the target minIO instance. Used by Enduro to read
+  contents of a bucket, or for any other minIO interactions.
+* `pathStyle`: Currently Amazon Web Services support two different URL
+  construction methods when interacting with an object store bucket via API. The
+  "path-style" method constructs the bucket's access URL using the configured
+  properties, such as region, bucket name, and object key. For Enduro
+  integrations, the second virtual "host-style" method is not currently
+  supported so if using an S3-like object store, ensure this is set to `true`.
+* `accessKey`: Username for accessing the configured S3-like bucket
+* `secretKey`: Password for accessing the configured S3-like bucket
+* `region`:  = AWS S3 buckets are created in a specific region. When interacting
+  with S3, you can specify the region during the bucket creation process. For
+  a full list of available regions and the syntax to specify them, consult the
+  [AWS S3 documentation][S3-regions].
+* `bucket`: A configured object store may have more than 1 bucket. This
+  parameter specifies the target bucket name to be used for the uploaded SIPs
+  and failed packages location. Because the default configuration uses MinIO
+  buckets elsewhere as well, ensure that this bucket name is unique.
+
+#### Internal storage configuration - Azure blob storage
+
+This subsection allows an [Azure] blob storage location to be configured for UI
+uploads and failed packages, instead of the default S3-like object store
+configured above. At installation the values in this subsection are left blank.
+
+If you intend to use Azure for your internal upload space, be sure to remove or
+comment out the S3-like object store parameters in the section
+[above](#internal-storage-configuration---s3-like-bucket-or-filesystem), and
+then add the Azure blob URL connection information there instead. For example,
+if your target Azure storage blob is named "sips", your configuration of the
+section above might look like this when configured properly:
+
+```toml
+[internalStorage.bucket]
+url = "azblob://sips"
+endpoint = ""
+pathStyle = ""
+accessKey = ""
+secretKey = ""
+region = ""
+bucket = ""
+```
+
+Meanwhile, this subsection provides Enduro with the connection information
+needed to access packages in the target Azure blob store.
+
+**Default values**:
+
+```toml
```

Comment:
I think it would be helpful to include the subsection header for context here: `[internalStorage.azure]`

Reply:
whoops, weird, not sure how i missed that. thanks